platformio.ini

```diff
@@ -24,16 +24,16 @@ lib_deps =
 
 [env:E1_coldflow]
 src_filter = -<*> +<E1_coldflow/>
-build_flags = -DHAL=teensy ; -DDEBUG
```

Comment:
undo these changes. we want to be able to easily uncomment, or comment out the DEBUG flag

Reply:
is this supposed to be grayed out @fazerlicourice7

Reply:
leave the `-DDEBUG` as a comment (the `;` is the comment identifier in`.ini` files)